tests/test_utils.py

```diff
@@ -1,6 +1,17 @@
+from inspect import isclass
+
 import pytest
 
-from scikeras._utils import pack_keras_model, route_params, unpack_keras_model
+from tensorflow.keras import losses as losses_module
+from tensorflow.keras import metrics as metrics_module
+from tensorflow.keras import optimizers as optimizers_module
+
+from scikeras._utils import (
+    compile_with_params,
+    pack_keras_model,
+    route_params,
+    unpack_keras_model,
+)
```

Comment:
Why are these imported?

Reply:
Because there are still tests using them. We can remove them as part of #79.